src/greyVest/utils.js

```diff
@@ -1,6 +1,7 @@
 import F from 'futil-js'
 
-export let openBinding = (...lens) => ({
-  isOpen: F.view(...lens),
-  onClose: F.off(...lens),
-})
+export let openBinding = lens =>
```

Comment:
We still need the spread here to support all the futil lens types, so with no args `lens` would be `[undefined]` instead of `undefined`. Maybe something like

```js
export let openBinding = (...lens) => 
  !_.isEmpty(lens) && _.head(lens) && {
    isOpen: F.view(...lens),
    onClose: F.off(...lens),
  }
```

Reply:
Ok just to summarize from our conversation: The current code fails to accoutn for lens being an array, but an array is not a valid lens right now. The lens prop API for view is `F.view(attr, obj)`, not `F.view([attr, obj])`, so unless we want to account for array lenses, this should be fine

Reply:
There's no way to pass two args as one so `lens` won't be `(prop, obj)`